tests/integration/components/app-notification-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('app-notification', 'Integration | Component | app notification', {
```

Comment:
See how other tests are defined and follow suit.

For reference: https://guides.emberjs.com/v2.13.0/testing/testing-components/